docs_user/modules/proc_adopting-networker-services-to-the-data-plane.adoc

```diff
@@ -2,34 +2,18 @@
 
 = Adopting Networker services to the {rhos_acro} data plane
 
-If you have `Networker` nodes in your existing {rhos_acro} deployment, you
-should adopt them to the {rhos_acro} data plane too. The procedure is largely
-the same as for `Compute` nodes.
+If you have Networker nodes in your existing {rhos_long} deployment, adopt them to the {rhos_acro} data plane. You decide which services you want to run on the Networker nodes, and then create a separate `OpenStackDataPlaneNodeSet` custom resource (CR) just for these nodes. You might also decide to implement the following options if they apply to your environment:
```

Comment:
I wrote the sentence “You decide which services you want to run on the Networker nodes, and then create a separate `OpenStackDataPlaneNodeSet` CR just for these nodes.” Is this sentence an accurate description of what the customer is doing in this procedure?

Reply:
I believe so, @slawqo / @booxter  do you agree?

Reply:
yes, it is correct. You can run there some networking specific services like e.g. dhcp agent in some cases, or just ovn-controller, etc.